components/server/ee/src/prebuilds/bitbucket-app.ts

```diff
@@ -35,7 +36,13 @@ export class BitbucketApp {
                 if (req.header("X-Event-Key") === "repo:push") {
                     const span = TraceContext.startSpan("BitbucketApp.handleEvent", {});
                     const secretToken = req.query["token"] as string;
+                    const event = await this.webhookEvents.createEvent({
+                        type: "push",
+                        status: "received",
+                        rawEvent: JSON.stringify(req.body),
+                    });
                     if (!secretToken) {
+                        await this.webhookEvents.updateEvent(event.id, { status: "dismissed_unauthorized" });
```

Comment:
I'd actually set `message: "No token provided"` here.

Reply:
going to improve error message in a separate PR.

BTW, this case is more of a "Broken Webhook registration", as it should provide a hint to what needs to be fixed.